### DISCUSSION TITLE
Gradle wrapper files are not updating during migrating to v4

### DISCUSSION THREAD

**Discussion by HarshaKosala** [ionic-team/capacitor#5990] [Q&A] [answered]
Hi during the migration from capacitor v3 to v3 the Gradle wrapper files don't get updated for some reason (refer to the image below).

![image](https://user-images.githubusercontent.com/19972882/195325853-04d03193-1ba7-4950-9035-1bc46a56b222.png)

Can this be neglected by simply upgrading the Gradle version using android studio? Or is there any other way of migration?

**Comment by jcesarmobile** [comment, marked as answer]
Having old gradle files can cause problems, but depends on how old they are, if you were in Capacitor 3 they are probably not very old. 
Can you run `cd android && ./gradlew wrapper --distribution-type all --gradle-version 7.4.2 --warning-mode all` and see if it works?
That's what the upgrade gradle step does.

**Reply by HarshaKosala** [reply]
Hi, this actually worked! Thanks!
